app/wama.web/Views/AdminConsole/Index.cshtml

```diff
@@ -5,11 +5,65 @@
 @using WAMA.Web.Model;
 
 @{
-
     ViewData[Constants.ADMIN_CONSOLE_ACTIVE_TOOL] = Constants.ADMIN_CONSOLE_HOME;
 }
 
-Todays activities
 
-<h1>Patron check-ins: 45</h1>
-<h1>Employee check-ins: 5</h1>
\ No newline at end of file
+<h1 class="panel"> Today's Activities</h1>
+<h2 class="panel" id="utc1"></h2>
+<head>
+
+    <title>Admin template</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+<body>
+    <div id="container">
+        <div id="box">
+            <div class="panel panel-primary">
+                <h4 class="panel panel-heading">Patron check-ins</h4>
+
+                <div class="panel-footer">
+                    Currently checked-in:
+                    <div class="large">45 </div>
+                </div>
+                <a href="#">
+                    <div class="panel-footer"> View Details</div>
+                </a>
+            </div>
+        </div>
+        <div id="box">
+            <div class="panel panel-primary">
+                <h4 class="panel panel-heading">Employee Check-ins</h4>
+                <div class="panel-footer">
+                    Currently checked-in: <div class="large">46</div>
+                </div>
+                <a href="#">
+                    <div class="panel-footer"> View Details</div>
+                </a>
+            </div>
+        </div>
+        <div id="box">
+            <div class="panel panel-primary">
+                <h4 class="panel panel-heading">Other Account Info</h4>
+                <div class="panel-footer">
+                    Accounts Pending Approval: <div class="large">47</div>
+                </div>
```

Comment:
Same here, also every other place on this file that uses the cs class you added.